addons/crm/tests/test_crm_lead_convert.py

```diff
@@ -446,7 +446,7 @@ def test_lead_convert_wizard_new_partner(self):
         test_partner_lead, test_partner_wizard, commercial_partner = self.env['res.partner'].create([
             {'name': 'Lead Test Partner'},
             {'name': 'Wizard Test Partner'},
-            {'name': 'Company Partner', 'is_company': True},
+            {'name': 'Company Partner'},
```

Comment:
Should have a VAT (does not seem to have a real impact on tests but to keep old intention at least)